packages/account/src/Sections/Profile/PersonalDetails/personal-details-form.tsx

```diff
@@ -366,27 +400,36 @@ const PersonalDetailsForm = observer(() => {
                                             name='phone'
                                             id={'phone'}
                                             label={localize('Phone number*')}
-                                            className={clsx({
-                                                'account-form__fieldset--phone':
-                                                    account_settings?.phone_number_verification?.verified,
-                                            })}
                                             //@ts-expect-error type of residence should not be null: needs to be updated in GetSettings type
                                             value={values.phone}
+                                            hint={hintMessage({
+                                                is_phone_number_editted: account_settings.phone !== values.phone,
+                                                is_phone_number_empty: !account_settings.phone,
+                                            })}
                                             onChange={(e: ChangeEvent<HTMLInputElement>) => {
                                                 handleChange(e);
                                                 setStatus('');
                                             }}
                                             onBlur={handleBlur}
                                             required
                                             error={errors.phone}
-                                            disabled={isFieldDisabled('phone') || is_phone_field_disable}
+                                            disabled={
+                                                isFieldDisabled('phone') ||
+                                                is_request_button_disabled ||
+                                                !!next_request_time
+                                            }
                                             data-testid='dt_phone'
                                         />
-                                        {isPhoneNumberVerificationEnabled &&
-                                            account_settings.phone &&
-                                            account_settings.phone === values.phone && (
-                                                <VerifyButton setIsPhoneFieldDisable={setIsPhoneFieldDisable} />
-                                            )}
+                                        {isPhoneNumberVerificationEnabled && (
+                                            <VerifyButton
+                                                is_verify_button_disabled={
```

Comment:
Shouldn't we disable the VerifyButton in all scenarios when the phone number field is disabled?